services/account_type/cloudtrail_owner.yaml

```diff
@@ -36,39 +36,15 @@ Resources:
         - Key: cloudzero-reactor-account-id
           Value: !Ref ReactorAccountId
 
-  SnsTopicPolicy:
```

Comment:
We don't need this Policy anymore, our future Reactor Lambdas can subscribe directly to SQS